FpsCheck.py

```diff
@@ -243,7 +243,12 @@ def is_eTLD_Plus1(self, site):
         Returns:
             boolean with truth value dependent on value of get_public_suffix
         """
-        return self.etlds.get_public_suffix(site, strict=True) is not None
+        assert site is not None
+        site = site.lstrip("https://")
+        is_etld_plus1 = self.etlds.get_public_suffix(site, strict=True) == site
+        is_not_etld = self.etlds.get_tld(site, strict=True) != site
+        return is_etld_plus1 and is_not_etld
```

Comment:
```suggestion
        is_etld_plus1 = self.etlds.get_public_suffix(site, strict=True) == site
        is_etld = self.etlds.get_tld(site, strict=True) == site
        return is_etld_plus1 and not is_etld
```

Optional: booleans are often clearer when written in the positive, so that we don't not never use double negatives. (See [go/tott/530](https://goto.google.com/tott/530), I don't think there's a public link.)